cpp/src/unary/null_ops.cu

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/cudf.h>
+#include <cudf/types.hpp>
+#include <cudf/utilities/type_dispatcher.hpp>
+#include <cudf/column/column_factories.hpp>
+#include <cudf/column/column_device_view.cuh>
+#include <cudf/detail/unary_detail.hpp>
+
+namespace cudf {
+
+namespace experimental {
+
+namespace detail {
+
+std::unique_ptr<column> null_op(column_view const& input,
```

Comment:
I think this could be made into a more generic utility function, something like `true_if`. 

```
template <typename InputIterator, typename Predicate>
unique_ptr<column> true_if(InputIterator begin, InputIterator end, Predicate p, rmm::mmr::device_memory_resource * mr, cudaStream_t stream);
```

This will return a column of `BOOL8` elements where every element `i` is true when `p(i)` returns true. 

So your usage could then become
```
return true_if(make_counting_iterator(0), make_counting_iterator(size), [device_view] __device__ (auto index) { return device_view.is_valid(index);});
```